RPG-game-Hung/classes/db.py

```diff
@@ -0,0 +1,18 @@
+def not_number(num):
+    """To validate if input is number or not"""
+    """Function is stored here to be able to import everywhere else"""
+    try:
+        int(num)
+    except ValueError:
+        print('Please enter only number\n')
+        return True
+    return False
+
+
+class Database:
```

Comment:
If this doesn't actually store the results to a real database, maybe rename it to something like `ResultChecker`